Documentation/gettingstarted_kafka.rst

```diff
@@ -0,0 +1,411 @@
+
+outpost-9999 terminal:
+::
+
+   $ OUTPOST_9999_POD=$(kubectl get pods -l outpostid=9999 -o jsonpath='{.items[0].metadata.name}') && kubectl exec -it $OUTPOST_9999_POD -- sh -c "PS1=\"outpost-9999 $\" /bin/bash"
+
+
+Step 4: Test Basic Kafka Produce & Consume
+==========================================
+
+First, let's have the *empire-hq* produce messages on both the *empire-announce* and *deathstar-plans*
+topics to confirm that data is flow as expected when everyone is behaving nicely.
+
+First, we will create the topics on the Kafka broker itself.  This is a one-time setup action that the
+operator of the Kafka cluster would perform.  From the main terminal, run:
+
+::
+
+   $ BROKER_POD=$(kubectl get pods -l app=kafka -o jsonpath='{.items[0].metadata.name}')
+
+   $ kubectl exec $BROKER_POD -c kafka sh -- -c "/opt/kafka/bin/kafka-topics.sh --create --zookeeper zook:2181 --replication-factor 1 --partitions 1 --topic empire-announce"
+     Created topic "empire-announce".
+
+   $ kubectl exec $BROKER_POD -c kafka sh -- -c "/opt/kafka/bin/kafka-topics.sh --create --zookeeper zook:2181 --replication-factor 1 --partitions 1 --topic deathstar-plans"
+     Created topic "deathstar-plans".
+
+First, let's start the consumer clients listening to their respective Kafka topics.  All of the consumer
+commands below will hang intentionally, waiting to print data they consume from the Kafka topic:
+
+In the *empire-backup* window, start listening on the top-secret *deathstar-plans* topic:
+
+::
+
+    $ ./kafka-consume.sh --topic deathstar-plans
+
+In the *outpost-8888* window, start listening to *empire-announcement*:
+
+::
+
+    $ ./kafka-consume.sh --topic empire-announce
+
+Do the same in the *outpost-9999* window:
+
+::
+
+    $ ./kafka-consume.sh --topic empire-announce
+
+Now from the *empire-hq*, first produce a message to the *empire-announce* topic:
+
+::
+
+   $ echo “Happy 40th Birthday to General Tagge” | ./kafka-produce.sh --topic empire-announce
+
+This message will be posted to the *empire-announce* topic, and show up in both the *outpost-8888* and
+*outpost-9999* windows who consume that topic.   It will not show up in *empire-backup*.
+
+*empire-hq* can also post a version of the top-secret deathstar plans to the *deathstar-plans* topic:
+
+::
+
+   $ echo “deathstar reactor design v3” | ./kafka-produce.sh --topic deathstar-plans
+
+This message shows up in the *empire-backup* window, but not for the outposts.
+
+Congratulations, Kafka is working as expected :)
+
+Step 5:  The Danger of a Compromised Kafka Client
+=================================================
+
+But what if a rebel spy gains access to any of the remote outposts that act as kafka clients?
+Since every client has access the Kafka broker on port 9092, it can do some bad stuff.
+For starters, the outpost container can actually switch roles from a consumer to a producer,
+sending “corrupting�� data to all other consumers on the topic.
```

Comment:
Something looks weird about the chars here? Is it suppose to be  â   ( âcorrupting) ?